.github/labels.yml

```
@@ -58,11 +61,17 @@
   color: "ff9efc"
 - name: "Complexity: 🟦 Easy to do"
   color: "ff9efc"
+  from_name: "good first issue"
 
 # Closing reason
 - name: "Closed: 👥 Duplicate"
   color: "959a9c"
   description: "Issue duplicates an existing issue"
+  from_name: "duplicate"
 - name: "Closed: ⚰️ Inactive"
   color: "959a9c"
   description: "No answer was received for weeks"
+- name: "Closed: 🙅 Won't fix"
+  from_name: "wontfix"
+- name: "Closed: 🧺 Invalid"
+  from_name: "invalid"
```

No need to migrate those labels since they are not tagging any issues/PRs yet + also adding the same color as other Closed reasons

```suggestion
- name: "Closed: 🙅 Won't fix"
  color: "959a9c"
- name: "Closed: 🧺 Invalid"
  color: "959a9c"
```